mainWidget/main_widget_FloorsAndRooms.yaml

```diff
@@ -1,139 +1,219 @@
 uid: main_widget_FloorsAndRooms
-tags: []
+tags:
+  - improvements Nico_R
 props:
   parameterGroups: []
-timestamp: Nov 25, 2022, 2:04:15 PM
+timestamp: Feb 22, 2023, 4:29:23 PM
 component: f7-block
 config:
-  style:
-    flex: 1 1 auto
-    margin-bottom: -1em
-    margin-top: 1em
-    overflow: scroll
-  visible: '=vars.objVar ? (vars.objVar.selectSection=="SECTION0") ? false : true : false'
+  stylesheet: >
+    .invisible {
+      display: none;
+      /*position: absolute;*/
+      /*left: -999em;*/
+      /*opacity: 0.1;*/
+    }    
 slots:
   default:
-    - component: oh-repeater
-      config:
-        fetchMetadata: semantics,metadata,widgetOrder
-        filter: '(vars.objVar && vars.objVar.floor) ? loop.floorItem.name==vars.objVar.floor : (loop.floorItem.metadata) && (loop.floorItem.metadata.semantics.value).includes("Floor")'
-        for: floorItem
-        itemTags: ","
-        sourceType: itemsWithTags
+    - component: f7-block
       slots:
```

Comment:
Visibility from line 13 needs to be readded her to prevent showing up on first start

Reply:
Is not necessary? All cols are hidden at initial start.
What's the point?

Reply:
I have to try again. With my first try, they all appeared. If this should not be the case, then I am fine with this.

Reply:
Not working, after clean start, cards are shown under weather widget without any selection.